src/atomization/atomizer.py

```diff
@@ -237,31 +242,35 @@ def delete_package_and_cleanup(package_id: int):
         logger.error(f"Database error: {e}")
         return False
 
+
 def sort_dafny_chunks(result: dict) -> list[dict]:
     """
     Takes a dictionary of categorized chunks and returns a flat list sorted by order
-    
+
     Args:
-        result: Dictionary with keys 'code', 'proof', 'spec', 'spec+code', 
+        result: Dictionary with keys 'code', 'proof', 'spec', 'spec+code',
                where each value is a list of dicts with 'content' and 'order' keys
-    
+
     Returns:
         List of dictionaries with 'content', 'order', and 'type' keys, sorted by order
     """
     # Create a flat list of all chunks with their types
     all_chunks = []
-    
+
     for chunk_type in ["code", "proof", "spec", "spec+code"]:
         for chunk in result.get(chunk_type, []):
-            all_chunks.append({
-                "content": chunk["content"],
-                "order": chunk["order"],
-                "type": chunk_type
-            })
-    
+            all_chunks.append(
```

Comment:
oh no. we seem to have different formatters on autosave. lol. 

I propose we unify around `ruff` (already in dev dependencies). you can run `uv run ruff format .` or `nix fmt` any time, or add `uv run ruff check .` to a precommit hook